### DISCUSSION TITLE
[solved] Control the air conditioning via MQTT with Tasmota

### DISCUSSION THREAD

**Discussion by SebastianStorb** [arendst/Tasmota#13188] [Q&A] [answered]
If I enter these commands in the Tasmota console, I can turn the air conditioning on and off:
```
IRhvac {"Vendor":"DAIKIN216","Power":"On","Mode":"Cool","FanSpeed":3,"Temp":18}
IRhvac {"Vendor":"DAIKIN216","Power":"Off","Mode":"Cool","FanSpeed":3,"Temp":18
```

I would like to know which command I have to send via MQTT so that one of these commands is executed in the Tasmota device.

I have set up the following MQTT structure in the Tasmota:

- tasmota/
--   Praxis/
----     Klimaanlage1/
-----       cmnd/
-----       stat/
-----       tele/
-----       cmnd/

I tried to send: 
`tasmota/Praxis/Klimaanlage1/cmnd/Power/IRhvac {"Vendor":"DAIKIN216","Power":"On","Mode":"Cool","FanSpeed":3,"Temp":18}`,`

or

`tasmota/Praxis/Klimaanlage1/cmnd/Power on IRhvac IRhvac {"Vendor":"DAIKIN216","Power":"On","Mode":"Cool","FanSpeed":3,"Temp":18}`
and many other things....

The result ist:
`16:34:34.695 MQT: tasmota/Praxis/Klimaanlage1/stat/RESULT = {"Command":"Unknown"}`

Could someone tell me how to do it right?

**Comment by barbudor** [comment, marked as answer]
Simply 
topic : `tasmota/Praxis/Klimaanlage1/cmnd/IRhvac`
payload `{"Vendor":"DAIKIN216","Power":"On","Mode":"Cool","FanSpeed":3,"Temp":18}`

`IRhvac` is a command, just like `power` or any other command
https://tasmota.github.io/docs/Commands/#with-mqtt

**Reply by SebastianStorb** [reply]
P E R F E C T - Thank U so much!!! It works

;-)))
